brave/brave-core
# feat(wallet): Unlock Wallet Page V2

## Description 
- Updates the `Unlock Wallet` page to the new `Wallet V2` designs


Resolves <https://github.com/brave/brave-browser/issues/31603>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to brave://wallet
2. You will see the new `Unlock Wallet` page design
3. Test and make sure everything works correctly
4. Enable `Enable Panel v2` flage in brave://flags
5. Open the panel, you should see the new `Unlock Wallet` Panel V2 design
6. Test and make sure everything works correctly

https://github.com/brave/brave-core/assets/40611140/cead83ed-8dc8-4a64-935e-8f4348d7976b

![Screenshot 66](https://github.com/brave/brave-core/assets/40611140/a2130aa5-eff5-4369-8b9b-955cb9af7843)

![Screenshot 65](https://github.com/brave/brave-core/assets/40611140/f06067e8-7916-4451-859e-e06181945bb4)
